.ci-operator.yaml

```diff
@@ -1,4 +1,4 @@
 build_root_image:
-  name: nodejs-16
```

Comment:
sorry, is `.ci-operator.yaml` used as build root image in CI? 
Based on [configuration](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/console-plugin-template/openshift-console-plugin-template-main.yaml)
```
build_root:
  image_stream_tag:
    name: release
    namespace: openshift
    tag: console-plugin-test-cypress
```
It looks like `.ci-operator.yaml` file is not used in CI because I don't see `from_repository: true` is specified
```
build_root:
  from_repository: true
```